scripts/check_frontend_coverage.py

```diff
@@ -347,6 +347,7 @@
     'svm-prediction.service.ts',
     'teacher.ts',
     'teacher2.ts',
+    'thumbnail-display.component.ts',
```

Comment:
I don't think we should be adding files here. Please add a test file. See the comment at the top of this list. 

If you need to add it here for some reason, please confirm with @nithusha21 before you do so.

Side note: @nithusha21 the comment at the top still refers to Mariana as the go-to person to check with before adding to the list. Should we change it?